Can’t run guide tests locally

The following is output after running `npm run test-guides`:

```
> donejs@1.0.0 test-guides /Library/WebServer/Documents/GitHub/donejs
> npm run test-quickstart && npm run test-pmo

npm WARN invalid config loglevel="notice"

> donejs@1.0.0 test-quickstart /Library/WebServer/Documents/GitHub/donejs
> guide guides/guide/test.js --local

Step 1: Remove existing dependencies
Step 2: Install donejs
npm ERR! addLocal Could not install /Library/WebServer/Documents/GitHub/donejs/undefined#master
npm ERR! Darwin 16.6.0
npm ERR! argv "/usr/local/bin/node" "/Library/WebServer/Documents/GitHub/donejs/node_modules/.bin/npm" "install" "undefined#master" "-g"
npm ERR! node v8.1.2
npm ERR! npm  v3.10.10
npm ERR! path /Library/WebServer/Documents/GitHub/donejs/undefined#master
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open

npm ERR! enoent ENOENT: no such file or directory, open '/Library/WebServer/Documents/GitHub/donejs/undefined#master'
npm ERR! enoent ENOENT: no such file or directory, open '/Library/WebServer/Documents/GitHub/donejs/undefined#master'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
```

[`/Library/WebServer/Documents/GitHub/donejs/` is where my local DoneJS repo is installed]

--- a/guides/guide/test.js
+++ b/guides/guide/test.js
@@ -27,7 +27,8 @@ guide.step("Remove existing dependencies", function(){
 guide.step("Install donejs", function(){
   var branch = process.env.TRAVIS_PULL_REQUEST_BRANCH || process.env.TRAVIS_BRANCH
     || process.env.APPVEYOR_REPO_COMMIT || 'master';
-  var repo = process.env.TRAVIS_PULL_REQUEST_SLUG || process.env.TRAVIS_REPO_SLUG;
+  var repo = process.env.TRAVIS_PULL_REQUEST_SLUG ||
+    process.env.TRAVIS_REPO_SLUG || "donejs/donejs";
 
 	return guide.executeCommand("npm", ["install", repo + "#" + branch, "-g"]);
 });

--- a/guides/place-my-order/test.js
+++ b/guides/place-my-order/test.js
@@ -40,7 +40,8 @@ guide.step("Remove existing dependencies", function(){
 guide.step("Install donejs", function(){
   var branch = process.env.TRAVIS_PULL_REQUEST_BRANCH ||
     process.env.TRAVIS_BRANCH || process.env.APPVEYOR_REPO_COMMIT || 'master';
-  var repo = process.env.TRAVIS_PULL_REQUEST_SLUG || process.env.TRAVIS_REPO_SLUG;
+  var repo = process.env.TRAVIS_PULL_REQUEST_SLUG ||
+    process.env.TRAVIS_REPO_SLUG || "donejs/donejs";
 
   return guide.executeCommand("npm", ["install", repo + "#" + branch, "-g"]);
 });

